multiqc/modules/fastp/fastp.py

```diff
@@ -366,9 +366,7 @@ def fastp_filtered_reads_chart(self):
         keys["filtering_result_passed_filter_reads"] = {"name": "Passed Filter"}
         keys["filtering_result_low_quality_reads"] = {"name": "Low Quality"}
         keys["filtering_result_too_many_N_reads"] = {"name": "Too Many N"}
-        keys["filtering_result_low_complexity_reads"] = {"name": "Low Complexity"}
-        keys["filtering_result_too_short_reads"] = {"name": "Too Short"}
-        keys["filtering_result_too_long_reads"] = {"name": "Too Long"}
+        keys["filtering_result_too_short_reads"] = {"name": "Too short"}
```

Comment:
Please do not make edits to other modules in this PR.

If you want to change those modules, do so in separate PRs (you can use different branches to keep them separate).

Reply:
Removed